scripts/clearwater-aio-install.sh

```diff
@@ -74,7 +74,7 @@ sed -e 's/#HEAP_NEWSIZE=".*"/HEAP_NEWSIZE="128M"/g' -i /etc/cassandra/cassandra-
 service cassandra stop
 
 # Install the remaining clearwater packages
-apt-get install -y --force-yes ellis bono restund sprout homer homestead homestead-prov
+apt-get install -y --force-yes ellis bono restund sprout homer homestead homestead-prov clearwater-prov-tools
```

Comment:
The manual command in the docs to install an AIO node is http://clearwater.readthedocs.org/en/latest/All_in_one_Images/index.html#manual-build, but that calls this script.

The `X-Start-Before` lines don't need updating - clearwater-prov-tools isn't "started" - it just consists of a set of tools.